content/events.json

```diff
@@ -0,0 +1,51 @@
+{
+  "liveEvents": [
+    {
+      "id": "event1",
+      "title": "Rock Night",
+      "image": "../pictures/rock.png",
+      "description": "Lokala rockband spelar hela natten lång hos Kino!",
+      "times": ["2025-01-15 20:00", "2025-03-22 20:00", "2025-08-03 20:00"]
```

Comment:
maybe "times" can be made as:
[
  {
     "date": "2025-01-15,
     "time": "20:00"
 }
] <br>instead?

Reply:
Thanks for your suggestion, I'll look into it :)